packages/studio-plugin/tests/writers/FileSystemWriter.test.ts

```diff
@@ -35,20 +29,10 @@ const bannerComponentState: ComponentState = {
   metadataUUID: "mock-metadata-uuid",
 };
 
-const moduleMetadata: ModuleMetadata = {
-  kind: FileMetadataKind.Module,
-  componentTree: [
-    {
-      kind: ComponentStateKind.Fragment,
-      uuid: "mock-uuid-parent",
-    },
-    {
-      ...bannerComponentState,
-      parentUUID: "mock-uuid-parent",
-    },
-  ],
-  metadataUUID: "metadata-uuid",
-  filepath: upath.join(paths.modules, "UpdatedModule.tsx"),
+const pageState: PageState = {
+  componentTree: [bannerComponentState],
+  cssImports: [],
+  filepath: upath.join(paths.pages, "UpdatedPage.tsx"),
 };
 
 describe("syncFileMetadata", () => {
```

Comment:
I don't think the test below is for `syncFileMetadata` anymore

Reply:
Very true!  I went ahead and changed that description and added a test to check writing to site settings.